examples/with-typescript-graphql/README.md

```diff
@@ -1,11 +1,11 @@
-# GraphQL and TypeScript Example
+# TypeScript and GraphQL Example
```

Comment:
Because it's "with-typescript-graphql", I thought the title should be like this.